webui/react/src/App.tsx

```diff
@@ -58,6 +59,7 @@ const AppView: React.FC = () => {
 
   useEffect(() => (isAuthenticated ? userStore.fetchCurrentUser() : undefined), [isAuthenticated]);
   useEffect(() => (isAuthenticated ? userStore.fetchUsers() : undefined), [isAuthenticated]);
+  useEffect(() => (isAuthenticated ? clusterStore.startPolling() : undefined), [isAuthenticated]);
```

Comment:
nit: since these all have the same dependency array, they could be combined

Reply:
each of those store calls `fetchUsers`, `fetchCurrentUser` and `startPolling` return a cleanup callback to get called upon an unmount. They can still be combined but the performance difference I believe is pretty negligible with a bit more lines of code. The alternative would look this:
```
useEffect(() => {
   const stopFetchCurrentUser = isAuthenticated ? userStore.fetchCurrentUser() : undefined;
   const stopFetchUsers = isAuthenticated ? userStore.fetchUsers() : undefined;
   const stopPolling = isAuthenticated ? clusterStore.startPolling() : undefined;
  return () => {
    stopFetchCurrentUser?.();
    stopFetchUsers?.();
    stopPolling?.();
  };
}, [isAuthenticated]);
```

Reply:
That makes sense, nevermind then.